test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -134,6 +136,16 @@ def test_network_transaction_scrubbing
     assert_scrubbed(@gateway.options[:secret_key], transcript)
   end
 
+  def test_store_transcript_scrubbing
+    response = nil
+    transcript = capture_transcript(@gateway) do
+      response = @gateway_api.store(@credit_card, @options)
+    end
+    token = response.responses.first.params['token']
```

Comment:
I'm getting this error when I run the tests:

`NoMethodError: undefined method 'responses' for #<ActiveMerchant::Billing::Response:0x00007f901c164b40>`

Reply:
Could you try again after the changes?